src/components/Person/PersonSpouse.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export const PersonSpouse = ({ sex, isMarried, spouse }) => {
+  const spouseType = sex === 'f' ? 'husband' : 'wife';
+  const text = isMarried ? `${spouse} is my ${spouseType}` : 'I am not married';
```

Comment:
[checkilist](https://github.com/mate-academy/react_person/blob/master/checklist.md): for better readability move ternary operator operands to separate line